src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Extensions/SourceTextExtensions.cs

```diff
@@ -269,4 +272,22 @@ public static bool TryGetSourceLocation(this SourceText text, int line, int char
         location = default;
         return false;
     }
+
+    public static TextEdit[] NormalizeTextEdits(this SourceText text, TextEdit[] edits)
+        => NormalizeTextEdits(text, edits, out _);
+
+    public static TextEdit[] NormalizeTextEdits(this SourceText text, TextEdit[] edits, out SourceText originalTextWithChanges)
```

Comment:
nit: A comment for these methods would be nice to explain a bit of what they're doing. Future me thanks you when I get an explanation in completion rather than having to look at the source to figure out if that's what I wanted to do

Reply:
You know what, I think I'll rename the method. I don't know that "normalizing" text edits makes sense, and I think one of the methods that duplicated this logic mentioned minimal text edits, which makes more sense. Thanks for the feedback

Reply:
New names and comments make much more sense. Thank you!